cmd/operator/main.go

```diff
@@ -241,6 +242,19 @@ func run(fs *flag.FlagSet) int {
 		return 1
 	}
 
+	canEmitEvents, _, err := k8sutil.IsAllowed(ctx, kclient.AuthorizationV1().SelfSubjectAccessReviews(), nil,
```

Comment:
can we capture the second returned value ([]error) and log the errors if the call succeeded but canEmitEvents is false?